README.md

```diff
@@ -96,6 +96,13 @@ When installing from source, you can choose to enable or disable a number
 of optional features using command line options to the `perl Makefile.PL`
 commands.
 
+### Ed25519
+
+Enabled by default.
+Disabled with `--no-ed25519`
+
+Requires support for Ed25519 in both openssl and ldns.
+
```

Comment:
Add:
```
> **Note:** The Zonemaster Engine test suite will, in some tests, give wrong result unless this feature
> is enabled.
```

Reply:
Very good point! I'll add something to that effect.